derp/derp_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Some time back there was a decision to only use https://github.com/frankban/quicktest in new code for test helpers when needed, but to avoid it when it didn't save much.

I'm not sure where/whether that's documented.